src/Console/ConfigurationResolver.php

```diff
@@ -258,15 +260,20 @@ public function getDiffer()
             $mapper = [
                 'null' => function () { return new NullDiffer(); },
                 'sbd' => function () { return new SebastianBergmannDiffer(); },
+                'udiff' => function () { return new UnifiedDiffer(); },
             ];
 
-            $option = $this->options['diff'] ? 'sbd' : 'null';
-
-            if (!isset($mapper[$option])) {
-                throw new InvalidConfigurationException(sprintf(
-                    'Differ must be "sbd" or "null", got "%s".',
-                    $this->options['diff']
-                ));
+            if ($this->options['diff-format']) {
```

Comment:
``--diff-format=udiff` without passing `--diff` will still produce diff ?

Reply:
yes it will, there is a utest for it

Reply:
weird

Reply:
yeah, well, I feel don't anything for one or the other, so happy to modify the PR in any way that makes it merge-able ;)

Reply:
are we fine with dropping the differ option on 3.0, leaving only udiff ?

Reply:
Depends if the people asking for `diff only` are OK with having the `udiff` format as proposed by this PR. If not, than it would be nice to keep the `differ-format` option on 3.0.
I'm not very happy with introducing `diff-format` here TBH, it is only done so we keep our BC promise.

Reply:
that/s exactly why I;m asking, to see what we want to have on 3,0 line.

Reply:
I'm not sure other formats will have actual value once true udiff will be available so I'm 👍 for dropping them in 3.0 along with the option, unless there are other useful formats we could add in the meantime.